Nos/Extensions/UIImage+File.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+import Logger
+
+extension UIImage {
+    
+    /// Initializes an image in memory from the file at the given URL.
+    convenience init?(fileURL: URL) {
```

Comment:
Is this function used in some place? I couldn't find an usage by just reading the code in Github.

Reply:
Nope! I used it in an earlier version but it's dead code now, so I removed it. Nice catch!